components/usage/pkg/apiv1/usage.go

```diff
@@ -24,10 +24,17 @@ import (
 
 var _ v1.UsageServiceServer = (*UsageService)(nil)
 
+type UsageConfig struct {
+	DefaultSpendingLimitForUsers int32 `json:"defaultSpendingLimitForUsers,omitempty"`
+
+	DefaultSpendingLimitForTeams int32 `json:"defaultSpendingLimitForTeams,omitempty"`
```

Comment:
```suggestion
	DefaultSpendingLimitForUsers int `json:"defaultSpendingLimitForUsers,omitempty"`

	DefaultSpendingLimitForTeams int `json:"defaultSpendingLimitForTeams,omitempty"`
```
This should be sufficient. `int` is actually compiler target dependant but often it's easier to use `int` and let the compiler deal with it.